app/ante.go

```diff
@@ -41,7 +43,8 @@ func NewAnteHandler(
 		ante.NewSetUpContextDecorator(), // outermost AnteDecorator. SetUpContext must be called first
 		wasmkeeper.NewLimitSimulationGasDecorator(wasmConfig.SimulationGasLimit),
 		wasmkeeper.NewCountTXDecorator(txCounterStoreKey),
-		ante.NewRejectExtensionOptionsDecorator(),
+		// UNFORKINGTODO C: I think this is correct in using nil for NewExtensionOptionsDecorator to reject all, but want ACK
+		ante.NewExtensionOptionsDecorator(nil),
```

Comment:
LGTM thanks for flagging

source https://github.com/cosmos/cosmos-sdk/blob/50f936f60e51f62fb34005d50c2c70efc94484b7/x/auth/ante/ext.go#L35-L41

Reply:
https://github.com/osmosis-labs/osmosis/pull/6413/commits/dbc2bde371fd6d077781084c33e319566bc55421